api/src/test/java/org/openmrs/api/ProviderServiceTest.java

```diff
@@ -520,4 +521,14 @@ private Person newPerson(String name) {
 		return person;
 	}
 	
+	@Test
+	public void createProviderFromUser_shouldGetUserAndCreateProvider() {
+		
+		User u = Context.getUserService().getUser(19901);
+		Provider p = Context.getProviderService().getProvider(1);
+		Assert.assertNotNull(Context.getPersonService().getPerson(19901));
+		Assert.assertNotNull(Context.getProviderService().getProvider(1));
```

Comment:
What does this test?

Reply:
This one tests the given person id that returns 19901, i provided the simple dataset